packages/web-components/fast-components/src/radio-group/fixtures/base.html

```diff
@@ -4,36 +4,46 @@ <h1>Radio Group</h1>
     <h4>Defaults</h4>
     <div>
         <fast-radio-group name="numbers">
-            <label style="color: ${neutralForegroundRestBehavior.var};" slot="label">Numbers</label>
+            <label style="color: ${neutralForegroundRestBehavior.var};" slot="label"
```

Comment:
Not yours, but I bet this interpolation doesn't work

Reply:
Hmmm, no I wouldn't think so

Reply:
Definitely not 🤣

Reply:
Filed an issue for it https://github.com/microsoft/fast/issues/3505